README.md

```diff
@@ -467,92 +465,59 @@ By default, the _scanner_ stores scan results on the local file system in the cu
 `~/.ort/scanner/scan-results`) for later reuse. Settings like the storage directory and the compression flag can be
 customized in the ORT configuration file (`-c`) with a respective storage configuration:
 
-```hocon
-ort {
-  scanner {
-    storages {
-      fileBasedStorage {
-        backend {
-          localFileStorage {
-            directory = "/tmp/ort/scan-results"
-            compression = false
-          }
-        }
-      }
-    }
-
-    storageReaders: [
-      "fileBasedStorage"
-    ]
-
-    storageWriters: [
-      "fileBasedStorage"
-    ]
-  }
-}
+```yaml
+ort:
+  scanner:
+    storages:
+      fileBasedStorage:
+        backend:
+          localFileStorage:
+            directory: "/tmp/ort/scan-results"
+            compression: false
+
+    storageReaders: ["fileBasedStorage"]
```

Comment:
Shouldn't this read as

    storageReaders:
    - "fileBasedStorage"

Similar below.

Reply:
YAML allows both and I used the shorter syntax to keep the examples more compact.

Reply:
Interesting. I wonder whether we then can / should use that more compact representation also in ORT result serialization...

Reply:
Jackson does not seem to provide an option for this:

https://github.com/FasterXML/jackson-dataformats-text/issues/43
https://fasterxml.github.io/jackson-dataformats-text/javadoc/yaml/2.13/com/fasterxml/jackson/dataformat/yaml/YAMLGenerator.Feature.html

Reply:
> YAML allows both and I used the shorter syntax to keep the examples more compact.

Note to myself: This is called [flow sequences](https://yaml.org/spec/1.2.2/#flow-sequences) in YAML. At least in SnakeYAML directly, which is what Jackson uses, there seems to be a (hacky) way to [configure flow vs. block style](https://stackoverflow.com/a/48281824/1127485).